### PR TITLE
chore: add new contributors :)

### PR DESCRIPTION
**Pull request by aladdin-add**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/qiankun/ISSUE_URL

### CODE REVIEW

File: contributors.json [hunk 1]
```diff
@@ -1 +1 @@
-["kuitos", "gongshun"]
+["kuitos", "gongshun", "aladdin-add<weiran.zsd@outlook.com>"]
```
Comment (kuitos) [inline contributors.json:1]: 现在 action 里面好像只匹配了用户名，我不确定带邮箱是不是可以

Comment (aladdin-add) [reply contributors.json:1]: 那我删掉 🤣

Comment (aladdin-add) [reply contributors.json]: 需要rebase么？

Comment (kuitos) [reply contributors.json]: 不用

Comment (aladdin-add) [reply contributors.json]: b17c0aa2ee6eab0d03a2f30572349f224bf0123f

**Review by kuitos — APPROVED** [review]
LGTM

**Pull request merged**